new/INS1023BadawoAragawi.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS1023BadawoAragawi" xml:lang="en" type="ins">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Badāwo ʾAbuna ʾAragāwi</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <t:change xmlns:t="http://www.tei-c.org/ns/1.0" who="JK" when="2025-04-14">Created entity</t:change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="dabr" subtype="institution">
+                    <placeName xml:id="n1" xml:lang="gez">በዳዎ፡ አቡነ፡ አረጋዊ፡</placeName>
+                    <placeName corresp="#n1" xml:lang="en">Badāwo ʾAbuna ʾAragāwi</placeName>
+                    <placeName xml:id="n2" xml:lang="gez">ደብረ፡ አረጋዊ፡</placeName>
+                    <placeName corresp="#n2" xml:lang="en">Dabra ʾAragāwi</placeName>
+                <note>
+                    <date type="foundation" from="1730" to="1755">Founded 
+                        by <persName ref="PRS14785EhetaMaryam"/> during the reign of <persName ref="PRS5617IyasuII"/> (1730–1755).</date>
+                </note>
+                </place>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="INS1023BadawoAragawi" passive="BDLaethd23#a4"/>
```

Comment:
BDLaethd23#a4 is correct, but I personally prefer to keep references in these cases to the general BDLaethd23 (the likelihood that addition numeration changes is quite high, but the likelihood that we remember to search other record types for this during the substitution is quite low)

Reply:
Ok :D I change it